src/context/chartDataContext.tsx

```diff
@@ -24,7 +36,8 @@ export const DataEndIndexContextProvider = DataEndIndexContext.Provider;
  *
  * @return data array for some charts and undefined for other
  */
-export const useChartData = () => useContext(ChartDataContext);
+export const useChartData = (): ChartData | undefined =>
```

Comment:
Brush is the only one that uses this hook and it is tested well so I am fairly confident this works just as before.